Runtime/Scripts/MediaStreamTrack.cs

```diff
@@ -108,6 +108,19 @@ internal VideoStreamTrack(string label, UnityEngine.Texture source, UnityEngine.
             m_destTexture = dest;
         }
 
+        /// <summary>
+        /// note:
+        /// The videotrack cannot be used if the encoder has not been initialized.
+        /// Do not use it until the initialization is complete.
+        /// </summary>
+        public bool IsInitialized
+        {
+            get
+            {
+                return WebRTC.Context.GetInitializationResult(self) == CodecInitializationResult.Success;
+            }
+        }
```

Comment:
https://github.com/Unity-Technologies/com.unity.webrtc/blob/4b2125547d6c23f66720e4e388a6a5ed7bd5cf3b/Runtime/Scripts/MediaStreamTrack.cs#L176
this method work another thread (not main thread)
so, this flag need check next flame

Reply:
Could you add test cases for the method ?

Reply:
I'll add that.
However, I don't think I can write a test because the maximum number of initialization failures depends on the GPU. Because there is no API to get the maximum number.

Reply:
You are right, we don't need the test case as you said.
But, at least we can test that the status works correctly
- `NotInitialized`: Not started the initialization of the encoder
- `Success`: the initialization of the encoder succeeds.

Reply:
I add assert about new method and property.

b03ddc2